### PR TITLE
[16.0][MIG] l10n_fr_fec_oca: Migration to 16.0

### PR DESCRIPTION
**Pull request by ghost**
The change of licence is due to the pylint/oca checks.

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi. could you take a look on the CI ? Thanks !

**Review by Camille0907 — APPROVED** [review]

File: l10n_fr_fec_oca/wizard/account_fr_fec_oca.py [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+            ("types", "Selected Account Types"),
+            ("accounts", "Selected Accounts"),
+            ("all", "All"),
+        ],
+        default="types",
+        required=True,
+        string="Partner Export Option",
+    )
+    partner_account_type_ids = fields.Many2many(
+        "account.account.type",
```
Comment (florian-dacosta) [inline l10n_fr_fec_oca/wizard/account_fr_fec_oca.py:73]: This model does not exist anymore.

**Review by florian-dacosta — CHANGES_REQUESTED** [review]

File: l10n_fr_fec_oca/wizard/account_fr_fec_oca.py (hunk 1 above)
Comment (legalsylvain) [reply l10n_fr_fec_oca/wizard/account_fr_fec_oca.py:73]: I can't understand how the CI can be green with such code line. Any idea ?

Comment (florian-dacosta) [reply l10n_fr_fec_oca/wizard/account_fr_fec_oca.py:73]: I am not a CI expert...you tell me ! :)
It just gives a warning on module installation, I guess the warnings are not a problem for the CI...
And there is not test on this module, else CI would have detected it for sure.

Comment (imlopes) [reply l10n_fr_fec_oca/wizard/account_fr_fec_oca.py:73]: @juliette-blanc Can you check the comments above ? (:

Comment (mourad-ehm) [reply l10n_fr_fec_oca/wizard/account_fr_fec_oca.py:73]: @juliette-blanc Thanks for this PR. Could you check the comment above ?

**Comment by metaminux** [comment]
Hello @juliette-blanc 

Are you still working on that PR ?

**Comment by legalsylvain** [comment]
/ocabot migration l10n_fr_fec_oca

Note : rebase should be done, because of conflict.

**Comment by alexis-via** [comment]
Here is the real serious work https://github.com/OCA/l10n-france/pull/459

**Comment by legalsylvain** [comment]
> Here is the real serious work https://github.com/OCA/l10n-france/pull/459

That's not the appropriate way to talk.

**Pull request closed without merging**